install_add_pod/after/Pods/Target Support Files/Pods-SampleApp/Pods-SampleApp-frameworks.sh

```diff
@@ -47,8 +47,13 @@ install_framework()
   local basename
   basename="$(basename -s .framework "$1")"
   binary="${destination}/${basename}.framework/${basename}"
+
   if ! [ -r "$binary" ]; then
     binary="${destination}/${basename}"
+  elif [ -L "${binary}" ]; then
+    echo "Destination binary is symlinked..."
```

Comment:
why is this echo-d?

Reply:
just for visibility purposes mostly, it has been the precent in this script before.

Reply:
see line 39 above